README.md

```diff
@@ -1,101 +1,147 @@
-<h1 align="center">
-    fzf.fish
-</h1>
+<div align="center">
 
-A plugin that integrates [fzf](https://github.com/junegunn/fzf) into your [fish](http://fishshell.com) workflow. Comes with handy functions&mdash;each with its own mnemonic key binding&mdash;that use fzf to
```

Comment:
I've confirmed that - renders to the actual emdash even though in the text editor it looks like a short dash.